src/plays/popup-component/PopupComponent.js

```diff
@@ -0,0 +1,36 @@
+import PlayHeader from 'common/playlists/PlayHeader';
```

Comment:
- Sort the import statements in alphabetical order for better.
- Use single quotes consistently for import statements.

import React from 'react';
import PlayHeader from 'common/playlists/PlayHeader';
import PopupCard from './components/popupCard';
import './styles.css';